recipes/moodycamel-concurrentqueue/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+from conans import ConanFile, CMake, tools
+from fnmatch import fnmatch
+import os
+
+
+class MoodycamelConcurrentqueueConan(ConanFile):
+    name = "moodycamel-concurrentqueue"
```

Comment:
```suggestion
    name = "concurrentqueue"
```

Follow the original project name